runtimes/dart-3.1/example/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: open_runtimes_example
+description: Dart runtime example
+version: 1.0.0
+publish_to: none
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
Does this need to be 3.1 ? @Meldiron

Reply:
Not too familiar with Dart release cycle, but this is what we do for 2.x versions: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/dart-2.15/pubspec.yaml#L8

We always od 2.12 - 3.0. I am guessing 2.12 had breaking change.

So until we see some breaking changes inside 3.x, we should be doing 3.0 to 4.0.

Reply:
If we limited this to exactly 3.0, then build will probably fail when developer uses some library that is for 3.1.
Which would be very annoying, and we would have to have every single version on Cloud to ensure all versions can be used with Appwrite.

I would rather address the problem when it pops up, disallowing versions that has breaking changes.